files/image_config/caclmgrd/caclmgrd

```diff
@@ -378,9 +378,7 @@ class ControlPlaneAclManager(object):
         # add iptables/ip6tables commands to drop all other incoming packets
         if num_ctrl_plane_acl_rules > 0:
             iptables_cmds.append("iptables -A INPUT -j DROP")
-            iptables_cmds.append("iptables -A FORWARD -j DROP")
```

Comment:
please also remove line 201 and 212

Reply:
Should we add forward rule accept here instead?

Reply:
@yxieca: That shouldn't be necessary as long as the default policy for the chain is ACCEPT, which is what is explicitly being done on lines 201 and 212.